odxtools/identvalue.py

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+from xml.etree import ElementTree
+
+from .exceptions import odxraise
+from .identvaluetype import IdentValueType
+from .odxdoccontext import OdxDocContext
+
+
+@dataclass(kw_only=True)
+class IdentValue:
+    """
+    Corresponds to IDENT-VALUE.
+    """
+
+    value: str
+
+    # note that the spec says this attribute is named "TYPE", but in
+    # python, "type" is a build-in function...
+    value_type: IdentValueType
```

Comment:
`type` should not be used as variable name, but could be used as an attribute without side effects
```suggestion
    type: IdentValueType
```

Reply:
I would prefer this as well, but `type()` is a build-in function.

Reply:
I'd like to avoid this anyway. If nothing else, it prevents confusing the syntax highlighting of IDEs. (BTW, the same argument can be made for `.odx_id` vs `.id`.)